### ISSUE TITLE
erusev/parsedown~1.3 breaks image uploads

### ISSUE THREAD

**Issue by mossadal** [rainlab/blog-plugin#47]
I just installed a fresh copy of october and all of a sudden the blog image upload stopped working. It took a while to track down the problem.

With parsedown~1.1.2, Markdown parses
```
![1](image)
```

as
```
<img alt="1" src="image" />
```
as expected by the TagProcessor filter.

When installing, I got parsedown~1.3 and now the Markdown parser returns
```
<img src="image" alt="1" />
```
thus breaking the regexp in the TagProcessor callback that replaces the slug `img` tag with the proper markup for Dropzone.

I haven't tracked down exactly which build of erusev/parsedown that changes the order of the attributes, but there have been a lot of new versions the last few days.

**Comment by Rhincodon** [comment]
Also on some markdown pages on hhvm — nFatal error: Call to undefined method DOMText::getAttribute() in /**/vendor/erusev/parsedown-extra/ParsedownExtra.php on line 446 
parsedown~1.3

**Comment by mossadal** [comment]
Rhincodon: I couldn't reproduce your issue. (But I'm not running hhvm, so that could be the reason.)

**Comment by daftspunk** [comment]
Fixed by #54
